src/fix_update_special_bonds.cpp

```diff
@@ -163,7 +197,37 @@ void FixUpdateSpecialBonds::pre_force(int /*vflag*/)
       }
     }
   }
+
+  for (auto const &it : new_created_pairs) {
+    tag1 = it.first;
+    tag2 = it.second;
+    i1 = atom->map(tag1);
+    i2 = atom->map(tag2);
+
+    // Loop through atoms of owned atoms i j and update SB bits
+    if (i1 < nlocal) {
+      jlist = firstneigh[i1];
+      jnum = numneigh[i1];
+      for (jj = 0; jj < jnum; jj++) {
+        j = jlist[jj];
+        if (j >> SBBITS & 3 != 0) continue;                  // Skip bonded pairs
```

Comment:
We should always use parentheses to indicate precedence in all but the most trivial if statements. In this case the precedence rules state that the "!=" is evaluated first which is probably not what you want.

```suggestion
        if (((j >> SBBITS) & 3) != 0) continue;                  // Skip bonded pairs
```

Reply:
Thanks Axel for spotting that, you are right. That was not what I intended.

Reply:
@jtclemm I didn't spot it, it was flagged by compiling with Clang++ 14 using -Wall -Wextra

Do you prefer to apply the necessary changes yourself, or do you want me to apply them? Or do them differently?

Reply:
I just added the change.